mpt/impl/imap-mailbox/postgres/pom.xml

```diff
@@ -117,11 +117,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <reuseForks>true</reuseForks>
-                    <forkCount>1C</forkCount>
                     <argLine>-Djava.library.path=
                         -javaagent:"${settings.localRepository}"/org/jacoco/org.jacoco.agent/${jacoco-maven-plugin.version}/org.jacoco.agent-${jacoco-maven-plugin.version}-runtime.jar=destfile=${basedir}/target/jacoco.exec
-                        -Xms512m -Xmx1024m -Dopenjpa.Multithreaded=true</argLine>
+                        -Xms1024m -Xmx2048m -Dopenjpa.Multithreaded=true</argLine>
```

Comment:
Really that's the problem?

Reply:
I saw a failure from "can not initial docker container", I'm not sure exactly. 
I see it configured differently in another module. So I tried.

Let me try with commit revert to `-Xms512m -Xmx1024m` and see what new

Reply:
I start to think the errors like `can not initial docker container` might be related to some specific workers that might have issues on the CI... I start keeping an eye on it

Reply:
I don't mind though, even if I don't believe it's the issue